xbmc/platform/darwin/osx/XBMCApplication.mm

```diff
@@ -97,11 +104,28 @@ - (void)applicationWillFinishLaunching:(NSNotification*)notification
                                                        keyEquivalent:@"t"];
   floatOnTopMenuItem.keyEquivalentModifierMask = NSEventModifierFlagCommand;
   [windowMenu addItem:floatOnTopMenuItem];
+
+  NSMenuItem* hideMenuItem = [[NSMenuItem alloc] initWithTitle:@"Hide"
+                                                        action:@selector(hideAppAction:)
+                                                 keyEquivalent:@"h"];
+  hideMenuItem.keyEquivalentModifierMask = NSEventModifierFlagCommand;
+  [windowMenu addItem:hideMenuItem];
+
   [windowMenu addItemWithTitle:@"Minimize"
                         action:@selector(performMiniaturize:)
                  keyEquivalent:@"m"];
 }
 
+- (BOOL)validateMenuItem:(NSMenuItem*)item
+{
+  // validate if the hide menu item should be shown
+  if ([item.title isEqual:@"Hide"] && [[NSApplication sharedApplication] isHidden])
```

Comment:
standard behavior is to have Show in place of Hide :)

Reply:
I've addressed both on a new commit. Just checked hide/show usually is shown in app submenu!
Thanks!